README.md

```diff
@@ -9,6 +9,7 @@ The following table lists the versions of Java available, and the parent Java im
 | 8             | eclipse-temurin:8-jammy  |
```

Comment:
is it time to remove 8 and 11?

Reply:
Definitely 8 can be removed but we can keep 11 for few more days. As part of next upgrade we will remove both 8 & 11